apps/mark-scan/backend/src/custom-paper-handler/scanner_status.ts

```diff
@@ -28,7 +28,7 @@ export function isPaperInScanner(
   );
 }
 
-export function isPaperInInput(
+export function isPaperPartiallyInInput(
```

Comment:
I don't care too much about naming, but IMO this rename is more confusing than previously, because if you told me the paper was partially in the input I would assume that some of the sensors are not firing, but the paper fully being in the input does trigger `isPaperPartiallyInInput`.

Reply:
Reverted naming!